src/main/java/it/pagopa/pn/deliverypush/service/impl/WebhookEventsServiceImpl.java

```diff
@@ -196,4 +202,44 @@ private Set<String> statusByVersion(int version) {
             .map(NotificationStatusInt::getValue)
             .collect(Collectors.toSet());
     }
+
+    private Set<String> categoriesByFilter(StreamEntity stream) {
+        Set<String> categoriesSet;
```

Comment:
Qui c'è da distinguere il comportamento rispetto alle API usate:
- per la V1 (e tale informazione ti arriva nell'header della versione), il comportamento rimane come prima (se filter values è vuoto, sottointende i soli eventi della v1)
- per la v2.3(+), il comportamento sarà invece quello di tenere conto appunto del placeholder. Il filtro vuoto invece lo userei in maniera simile a quanto fatto alla v1, ma usando la versione dello stream, in modo da tornare solo gli eventi per quella versione. Di per sè, avendo ora "resettato" il tutto alla 2.3, non servirebbe farci logica sopra adesso (se c'è tempo la implementiam e la aggiungo all'SRS se non era specificata).

Reply:
Le modifiche richieste sono state effettuate.

Reply:
credo che il codice si possa semplificare avendo:
webhookUtils.getVersion che ritorna "10", se lo stream è vuoto o vale "v10" (ma in questo secondo caso rientra nel normale parse della versione)

qui averere semplicemente 
```
if (CollectionUtils.isEmpty(stream.getFilterValues()){
          categoriesSet = categoriesByVersion(webhookUtils.getVersion(stream.getVersion()));
} else {
            categoriesSet = stream.getFilterValues().stream()
                    .filter(v -> !v.equalsIgnoreCase(DEFAULT_CATEGORIES))
                    .collect(Collectors.toSet());
            if (stream.getFilterValues().contains(DEFAULT_CATEGORIES)) {
                categoriesSet.addAll(pnDeliveryPushConfigs.getListCategoriesPa());
            }
        }
```